pkg/cmd/propertiesDelete.go

```diff
@@ -34,14 +34,16 @@ var (
 
 func init() {
 	parentCommand := propertiesCmd
-	parentCommand.MarkPersistentFlagRequired("name")
-	parentCommand.MarkPersistentFlagRequired("namespace")
 	parentCommand.AddCommand(propertiesDeleteCmd)
 }
 
 func executePropertiesDelete(cmd *cobra.Command, args []string) {
 	var err error
 
+	err = validateInputsAreNotEmpty(namespace, propertyName)
```

Comment:
Noooo. Try to not add code to this function, as it can't be unit tested currently.

Add it to the DeleteProperty call.